### PR TITLE
Gift Entry: Display options for template sections

### PR DESCRIPTION
**Pull request by voduyemi**
- Form template sections in the Template Builder now have display options for being expanded or collapsed when entering a gift in a single gift entry form or batch gift entry form

- Custom Labels
1. geHeaderFormFieldsModalDisplayOptionsSection
2. geLabelDisplaySections
3. geButtonFormFieldsDisplayOptionExpanded
4. geButtonFormFieldsDisplayOptionCollapsed

# Critical Changes

# Changes
## Gift Entry Updates
- Users can now set a section in a template as expanded or collapsed by default.
# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by voduyemi** [comment]
**lurch: attach W-038036

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HpexQAC" target="_blank">W-038036</a>

**Comment by melissabarber** [comment]
@voduyemi and @ptewson-sfdo I've checked the labels in this PR, updated the description to get it ready for release notes, and we're all good to go. Thanks!

**Comment by voduyemi** [comment]
@vmartinov-sforg That's interesting, it should work fine, works fine in my org, but I will try deploying to a fresh scratch org

File: src/lwc/geTemplateBuilder/geTemplateBuilder.js [hunk 1]
```diff
@@ -333,7 +333,7 @@ export default class geTemplateBuilder extends NavigationMixin(LightningElement)
         if (modalData.action === SAVE) {
             let formSections = mutable(this.formSections);
             let formSection = formSections.find((fs) => { return fs.id === modalData.section.id });
-
+            formSection.defaultDisplayMode = modalData.section.defaultDisplayMode;
```
Comment (voduyemi) [inline src/lwc/geTemplateBuilder/geTemplateBuilder.js:336]: @vmartinov-sforg This was the problem, I mistakenly reverted this line so the `defaultDisplayMode` was not getting saved to the Template JSON. It is fixed now.

File: src/lwc/geFormSection/geFormSection.js [hunk 2]
```diff
@@ -22,6 +22,11 @@ export default class GeFormSection extends LightningElement {
         return 'Toggle ' + this.section.label;
     }
 
+
+    get shouldCollapseSection() {
```
[the remark this answers is not in the document]
Comment (voduyemi) [reply src/lwc/geFormSection/geFormSection.js]: Updated!

**Pull request merged**